test/test_helper.rb

```diff
@@ -8,15 +8,26 @@
 require 'mocha/mini_test'
 
 require 'vagrant-service-manager'
-# capibilities
 require 'guests/redhat/cap/box_version'
 require 'guests/redhat/cap/os_variant'
-require_relative 'support/fake_ui'
 
 def fake_environment(options = { enabled: true })
   { machine: fake_machine(options), ui: FakeUI }
 end
 
+class FakeUI
+  attr_reader :received_info_messages
+
+  def initialize
+    @received_info_messages = []
+  end
+
+  def info(*args)
+    # puts "#{args}"
```

Comment:
Because it would print into the default progress bar of the test run. I really like to keep this clean. On the other hand, when debugging it is sometimes handy to turn on. Hence, I left in in there commented out.